ui/app/templates/components/oidc/assignment-form.hbs

```diff
@@ -0,0 +1,82 @@
+<div ...attributes>
+  <PageHeader as |p|>
+    <p.top>
+      <nav class="breadcrumb" aria-label="breadcrumbs">
+        <ul>
+          <li>
+            <span class="sep">&#x0002f;</span>
+            {{#if @model.isNew}}
+              <LinkTo @route={{"vault.cluster.access.oidc.assignments"}}>
+                Assignments
+              </LinkTo>
+            {{else}}
+              {{! You're editing in this view }}
+              <LinkTo @route={{"vault.cluster.access.oidc.assignments.assignment.details"}} @model={{@model.name}}>
+                Details
+              </LinkTo>
+            {{/if}}
+          </li>
+        </ul>
+      </nav>
+    </p.top>
+    <p.levelLeft>
+      <h1 class="title is-3" data-test-oidc-assignment-title>
+        {{if @model.isNew "Create" "Edit"}}
+        assignment
+      </h1>
+    </p.levelLeft>
+  </PageHeader>
+
+  <form {{on "submit" (perform this.save)}}>
+    <div class="box is-sideless is-fullwidth is-marginless">
+      <FormFieldLabel for="name" @label="Name" />
+      <input
+        autocomplete="off"
+        spellcheck="false"
+        value={{@model.name}}
+        disabled={{not @model.isNew}}
+        class="input field {{if this.errors.name.errors 'has-error-border'}}"
+        {{on "input" this.handleOperation}}
+      />
+      {{#if this.errors.name.errors}}
+        <AlertInline @type="danger" @message={{join ", " this.errors.name.errors}} />
+      {{/if}}
+
+      <SearchSelect
```

Comment:
Since the default is `disallowNewItems=false`, so by default users are allowed to create new groups and/or entities. Probably want to check with design that this is what we want! It's hard to tell from the figma file